src/App.php

```diff
@@ -157,7 +165,7 @@ public function initLayout($layout, $options = [])
         }
         $layout->app = $this;
 
-        $this->html = new View(['defaultTemplate'=>'html.html']);
+        $this->html = new View(['defaultTemplate' => $this->defaultTemplate]);
```

Comment:
application doesn't really have a template, as it's not extending View. Having defaultTemplate would be confusing here. If possible leave it 'html.html' for now as I don't want people to tamper with this too much at this point.